### DISCUSSION TITLE
Режим обслуживания.

### DISCUSSION THREAD

**Discussion by ivev7** [totumonline/totum-ru-issues-and-discussions#95] [Есть идея, а что если...]
Как перевести тотум в режим обслуживания , т.е. чтоб пользователи не могли зайти и видели картинку с надписью "СЕРВИСНЫЙ РЕЖИМ"?

**Comment by ttm-alexeypolunin** [comment]
Записал в пожелания. Не раньше середины Мая.

Сейчас можно всем пользователям (кроме админа конешн) снять галочку в users, но таблички «на обслуживании» не будет, но всех вышибет при следующем действии и не даст авторизоваться пока польз отключены.

**Reply by ivev7** [reply]
В настройках кнопка "РЕЖИМ ОБСЛУЖИВАНИЯ" , смотрелась бы прелестно!

**Comment by erpic-ru** [comment]
Кажется, что задачу можно решить на уровне редактирования .htaccess: принудительно переадресовывать запросы со всех ip (кроме ip админа) на страничку-сообщение об ограничении доступа и технических работах. Как-то так:

```

RewriteCond %{REMOTE_ADDR} !=37.122.8.243
RewriteRule ^.* maintenance.php [L]
```

**Comment by ivev7** [comment]
Спасибо
